modules/ROOT/pages/api-led-develop.adoc

```diff
@@ -0,0 +1,121 @@
+  .. Click the green plus sign to create a global configuration for HTTP listeners in this project.
+  .. Verify that the host is `0.0.0.0` and the port is `8081`.
+  .. Click *Test Connection*, and then click *OK* when the connection is successful.
+  .. If you click *Global Elements* at the bottom of the canvas, you'll see this configuration is a global configuration.
+. Click *Configuration XML* at the bottom of the canvas to see the XML that has been created.
+
+Once we have an implementation set up, we'll return to `interface.xml`.
+
+=== Step 4: Create the RESTful API Implementation
+
+Find and assemble the elements needed to define the business logic of your API:
+
+. In Studio, select *File > New > Mule Configuration File*.
+. Select the project `hello-world`, and then name the file `implementation.xml`.
+. Click *Finish*.
+. Add an HTTP connector endpoint to receive requests:
+  .. In the *Mule Palette*, select *HTTP*, the HTTP module.
+  .. Drag *Listener* from the *Mule Palette* to the canvas. This action creates a flow containing the listener.
+. Specify the global properties that the Listener needs:
+  .. Double-click HTTP Listener.
+  .. In the General tab of the Listener properties, notice that the global configuration that you created in Step 3 is displayed. If it is not, select it.
+  .. Set *Path* to */greeting*, the resource we specified in xref:api-led-design.adoc[Design an API specification].
+  .. Save your changes.
+. Specify logic to display the data from Hello World:
+  .. Return to the *Mule Palette* and select *Favorites* or *Core* to display the *Set Payload* transformer.
+  .. Find and drag *Set Payload* into the flow and leave it selected.
+  .. In the *General* tab, set *Value* under *Settings* to *Today the greeting is Hello.*
+. Save your work, and then review the XML generated by your work in *Configuration XML*, and then return to *Message Flow*.
+
+You have created a simple Mule app with an interface and an implementation.
+Studio provides a Mule runtime engine wher you can deploy and run your app for testing.
+
+=== Step 5. Test the API
+
+Run the Studio project that contains your API in the Mule runtime engine and then test with a third-party client:
+
+. Right-click anywhere on the canvas and select *Run project hello world*.
+. The console opens and displays status messages as the app is deployed to the Mule runtime engine (Mule) contained within Studio.
++
+When the following message is displayed, the app containing your API has been deployed to Mule.
+. To exercise the app, open a REST client (Postman) and issue a GET request:
+  `\http://localhost:8081/greeting`.
+. Your REST client should return a `200 OK` message and the body `Today the greeting is Hello.`.
+. Right-click in the canvas and select *Stop project hello-word*.
+
+== What's Next
+
+Add validation, event logging to support debugging, error handling, and a unit test to your API.
```

Comment:
Because it is a longish list, how about
```suggestion
Add to your API validation, event logging to support debugging, error handling, and a unit test.
```

Reply:
Great catch! made it a list.
done.